cypress/integration/pages/mostReadPage/tests.js

```diff
@@ -0,0 +1,41 @@
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`No testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    describe(`Tests for ${service} ${pageType}`, () => {
+      it('should have a visually hidden top-level header', () => {
+        cy.get('h1').should('have.length', 1);
+      });
+
+      it('should have mostread component rendered ', () => {
+        cy.get('main').should('have.attr', 'data-e2e', 'most-read');
+      });
+      if (Cypress.env('APP_ENV') !== 'local') {
+        it('should have articles list each linking to article page', () => {
```

Comment:
Is this test checking the first link in the most read list and checking if it doesn't 404?

Reply:
yeah exactly.

Reply:
Do we need to do this in the cypress tests? Is it enough to have an integration test that ensures that the links are rendered?

Reply:
I agree, I don't think we need this test. I will remove shortly.